README.md

```diff
@@ -144,6 +144,7 @@ You can pass the following options to `cmake ..`:
 
 * `-DBUILD_SAMPLE` -- Build the sample executables. ON by default.
 * `-DIOT_CORE_ENABLE_CREDENTIALS` -- Build the sample applications using IoT credentials. OFF by default.
+* `-DENABLE_DATA_CHANNEL` -- Build SDK & samples with data channel. ON by default.
```

Comment:
Is there anything that would break implicitly when it is turned off?

Reply:
https://github.com/search?q=repo%3Aawslabs%2Famazon-kinesis-video-streams-webrtc-sdk-c%20ENABLE_DATA_CHANNEL&type=code

Doesn't seem like it. We can add this combination to the CI in a future pr.